includes/user/User.php

```diff
@@ -552,7 +568,7 @@ function ( $oldValue, &$ttl, array &$setOpts ) use ( $cache ) {
 
 				return $data;
 			},
-			[ 'pcTTL' => $cache::TTL_PROC_LONG, 'version' => self::VERSION ]
+			$cacheOpts
```

Comment:
Can we also wrap this line in `Fandom change` comment block

Reply:
I guess this is vissible when looking at the first change so I don't think that this is necessary. But yeah we can add that here too 🙂